typescript/packages/plugins/coingecko/src/parameters.ts

```diff
@@ -39,3 +39,49 @@ export class SearchCoinsParameters extends createToolParameters(
         exact_match: z.boolean().optional().default(false).describe("Only return exact matches for the search query"),
     }),
 ) {}
+
+export class GetTokenPriceParameters extends createToolParameters(
+    z.object({
+        id: z.string().describe("Asset platform's id (e.g., 'ethereum')"),
+        contractAddresses: z.string().describe("The contract address of tokens (comma-separated)"),
```

Comment:
Make it an array, format the comma separated in the function that uses this instead